src/css/mixins.scss

```diff
@@ -2,3 +2,102 @@
   height: 100%;
   width: 100%;
 }
+@mixin xs {
```

Comment:
These mixins looks great! Tested them out with coloring to see how it changes at the different break points and everything resizes so fluidly!